lib/babe/build_integration_test.go

```diff
@@ -278,10 +236,12 @@ func TestBuildAndApplyExtrinsic(t *testing.T) {
 	require.Equal(t, []byte{0, 0}, res)
 }
 
+// TODO investigate if this is a needed test
```

Comment:
reminder to remove or address this

Reply:
do we have any other tests that assert an error returned from `buildBlock`?  If not, it makes sense to cover some error case.

Reply:
Don't believe so. I also think this test can be cleaned up. Its on my list of ones to address in my follow up PR